.github/workflows/pr-ci.yml

```diff
@@ -73,13 +82,25 @@ jobs:
         registry: ghcr.io/${{ github.repository_owner }}
         username: ${{ github.event.pull_request.user.login }}
         password: ${{ secrets.GHCR_PR_TOKEN }}
-      if: github.repository_owner == 'cryostatio'
+    - name: store image amd64
+      id: amd64_image
+      outputs:
+        amd64_image: ${{ steps.push-to-ghcr.outputs.registry-path }}
+      if: matrix.arch == 'amd64'
+    - name: store image arm64
+      id: arm64_image
+      outputs:
+        amd64_image: ${{ steps.push-to-ghcr.outputs.registry-path }}
+      if: matrix.arch == 'arm64'
     - name: Comment test image link
       uses: thollander/actions-comment-pull-request@v1
       with:
         message: |-
-          Test image available:
+          arm64 test image available:
+          ```
+          $ CRYOSTAT_IMAGE=${{ steps.amd64_image.outputs.amd64_image }} sh smoketest.sh
           ```
-          $ CRYOSTAT_IMAGE=${{ steps.push-to-ghcr.outputs.registry-path }} sh smoketest.sh
+          amd64 test image available:
           ```
-      if: github.repository_owner == 'cryostatio'
+          $ CRYOSTAT_IMAGE=${{ steps.arm64_image.outputs.arm64_image }} sh smoketest.sh
```

Comment:
I personally prefer having the command with no `$` shell prompt, because then we can just copy the command without removing the $ later.

Reply:
How is https://github.com/cryostatio/cryostat/pull/1599#discussion_r1282457303?

Reply:
Makes sense!